### ISSUE TITLE
Resolving of absolute paths with leading res:// and without result in different final paths

### ISSUE THREAD

**Issue by chanon** [godotengine/godot#4280]
**Operating system or device:**

Windows 10 64 bit, in the editor.

**Issue description** (what happened, and what was expected):

```
const MyScript = preload("res://path/to/MyScript.gd")
const MyScript2 = preload("/path/to/MyScript.gd")

# later

print(MyScript.get_path())
print(MyScript2.get_path())
print(MyScript == MyScript2)
```

Expected output:

```
res://path/to/MyScript.gd
res://path/to/MyScript.gd
True
```

Actual output:

```
res://path/to/MyScript.gd
res:///path/to/MyScript.gd
False
```

**Comment by chanon** [comment]
If using `preload("/path/to/MyScript.gd")` style path is not recommended / not supported then there should be a warning as suggested in https://github.com/godotengine/godot/issues/3106

**Comment by akien-mga** [comment]
@Faless Fixed by #6702?

**Comment by Faless** [comment]
> @Faless Fixed by #6702?

@akien-mga  Yes, it actually throws an error instead of just a warning, so users don't get confused. You can close this and also close #3106
